gdcdictionary/schemas/germline_mutation_calling_workflow.yaml

```diff
@@ -44,6 +51,10 @@ properties:
   workflow_type:
     term:
       $ref: "_terms.yaml#/workflow_type"
-    enum: [ "HaplotypeCaller" ]
+    enum:
+      - "HaplotypeCaller"
+      - "FreeBayes"
```

Comment:
For consistency, the quotes aren't really necessary. GDCAPI will cast them as a string.